app/models/case/base.rb

```diff
@@ -83,7 +83,7 @@ def self.searchable_document_tsvector
       .order(Arel.sql("(cases.properties ->> 'external_deadline')::timestamp with time zone ASC, cases.id"))
   }
   scope :by_last_transitioned_date, -> { reorder(last_transitioned_at: :desc) }
-  scope :most_recent_first, -> {reorder(Arel.sql("(cases.properties ->> 'external_deadline')::timestamp with time zone DESC, cases.id")) }
+  scope :most_recent_first, -> {reorder(Arel.sql("(properties ->> 'external_deadline')::timestamp with time zone DESC, cases.id")) }
```

Comment:
Is reverting the `cases.properties` on purpose or was just a rebase conflict? Just in case it was a mistake! 🤔